recipe/meta.yaml

```diff
@@ -42,6 +42,7 @@ requirements:
 test:
   commands:
     - test -f $PREFIX/lib/libopenslide${SHLIB_EXT}  # [unix]
+    - openslide-show-properties --help  # [unix]
```

Comment:
Instead, I'd suggest `env OPENSLIDE_DEBUG=synthetic openslide-show-properties ""` (using the legacy command-line tool) or `env OPENSLIDE_DEBUG=synthetic slidetool prop list ""` (using its replacement).  That runs an internal self-test which exercises every dependency.

Reply:
```suggestion
    - env OPENSLIDE_DEBUG=synthetic slidetool prop list ""  # [unix]
```